api/logic/modplatform/modpacksch/FTBPackInstallTask.cpp

```diff
@@ -174,6 +175,59 @@ void PackInstallTask::install()
     instance.setIconKey(m_instIcon);
     instanceSettings->resumeSave();
 
+    // flag certain mods
+    QDir modsDir(FS::PathCombine(m_stagingPath, "minecraft", "mods"));
+    if (modsDir.exists()) {
+        QVector<ModpacksCH::FlaggedMod> flaggedMods;
+
+        for (const auto& info : modsDir.entryInfoList(QDir::NoDotAndDotDot | QDir::Files)) {
+            // FTB Auxilium
+            if (info.fileName().startsWith("ftbauxilium") && info.fileName().endsWith(".jar")) {
+                flaggedMods.append(ModpacksCH::FlaggedMod {
+                    info.absoluteFilePath(),
+                    "FTB Auxilium",
+                    tr("Sends system information, crash reports, and when you join/quit singleplayer or multiplayer sessions to FTB's servers."),
+                    tr("Tracking")
+                });
+            }
+
+            // MCMC
+            if (info.fileName().startsWith("mcmc") && info.fileName().endsWith(".jar")) {
```

Comment:
This is very vague.

Reply:
@jamierocks said the "filter by name" will be replaced by a "filter by mod ID". The "filter list" might not be part of MultiMC but externalized like "meta".